httpobs/scanner/analyzer/headers.py

```diff
@@ -152,6 +206,8 @@ def cookies(reqs: dict, expectation='cookies-secure-with-httponly-sessions') ->
                 'cookies-session-without-httponly-flag',
                 'cookies-session-without-secure-flag']
 
+    # TODO: Support cookies set over http-equiv (ugh)
```

Comment:
File a bug for this and add the URL to it here?

Reply:
Done, see #265.